Sources/VaporRedis/RedisCache.swift

```diff
@@ -1,74 +1,72 @@
 import Redbird
 import Cache
 import Node
+import JSON
+import Core
 
 /**
-    Uses an underlying Redbird
-    instance to conform to the CacheProtocol.
-*/
+ Uses an underlying Redbird
+ instance to conform to the CacheProtocol.
+ */
 public final class RedisCache: CacheProtocol {
     /**
-        The underlying Redbird instance.
-    */
+     The underlying Redbird instance.
+     */
     public let redbird: Redbird
-
+    
     /**
-        Creates a RedisCache from the address,
-        port, and password credentials.
-
-        Password should be nil if not required.
-    */
+     Creates a RedisCache from the address,
+     port, and password credentials.
+     
+     Password should be nil if not required.
+     */
     public convenience init(address: String, port: Int, password: String? = nil) throws {
         let config = RedbirdConfig(address: address, port: UInt16(port), password: password)
         let redbird = try Redbird(config: config)
         self.init(redbird: redbird)
     }
-
+    
     /**
-        Create a new RedisCache from a Redbird.
-    */
+     Create a new RedisCache from a Redbird.
+     */
     public init(redbird: Redbird) {
         self.redbird = redbird
     }
-
+    
     /**
-        Errors that can be thrown by the RedisCache.
-        Other errors from the underlying Redbird
-        instance can also be thrown.
-    */
+     Errors that can be thrown by the RedisCache.
+     Other errors from the underlying Redbird
+     instance can also be thrown.
+     */
     public enum Error: Swift.Error {
         case incompatibleValue
     }
-
+    
     /**
-        Returns a key from the underlying Redbird 
-        instance by using the GET command.
-    */
-	public func get(_ key: String) throws -> Node? {
+     Returns a key from the underlying Redbird
+     instance by using the GET command.
+     */
+   	public func get(_ key: String) throws -> Node? {
         guard let result = try redbird.command("GET", params: [key]).toMaybeString() else {
             return nil
         }
-
-        return Node.string(result)
-	}
-
+            
+        return try JSON(bytes: result.bytes).makeNode()["data"]?.node
+    }
+    
     /**
-        Sets a key to the supplied value in the
-        underlying Redbird instance using the
-        SET command.
-    */
-	public func set(_ key: String, _ value: Node) throws {
-        guard let string = value.string else {
-            throw Error.incompatibleValue
-        }
-
-        try redbird.command("SET", params: [key, string])
-	}
-
+     Sets a key to the supplied value in the
+     underlying Redbird instance using the
+     SET command.
+     */
+    public func set(_ key: String, _ value: Node) throws {
+        try redbird.command("SET", params: [key, JSON(node: Node(["data": value])).serialize().toString()])
```

Comment:
should be able to use `node: ["data": value]` directly w/ the external param there, let me know if you're not seeing that.